android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/GlobalPreferences.kt

```diff
@@ -35,7 +35,7 @@ data class GlobalPreferences(
         var diskInterval: Double = 0.0,
         var diskMaxUsedGB: Double = 0.0,
         var diskMaxUsedPct: Double = 0.0,
-        var diskMinFreeGB: Double = 0.0,
+        var diskMinFreeMB: Double = 0.0,
```

Comment:
Are you sure it should be `MB` not `GB` ? Because it refers to `GlobalPreferences.Fields.DISK_MIN_FREE_GB`